src/main/java/com/prgrms/be/intermark/domain/schedule_seat/dto/ScheduleSeatFindRequestDTO.java

```diff
@@ -0,0 +1,9 @@
+package com.prgrms.be.intermark.domain.schedule_seat.dto;
+
+import javax.validation.constraints.NotNull;
+
+import lombok.Builder;
+
+@Builder
```

Comment:
intelij 에서 record에 builder 동작 하나요? 전 안되길래 이렇게 했는데...
```suggestion
public record ScheduleSeatFindRequestDTO(@NotNull Long scheduleId) {
@Builder public ScheduleSeatFindRequestDTO {}
```